submissions/morenko/oop-exercise/index.js

```diff
@@ -0,0 +1,75 @@
+// создадим родительский класс WorldInhabitant
+class WorldInhabitant {
+  // объявляем метод конструктор (ожидаем на входе значения, список свойств жителя)
+  constructor(name, species, gender, saying) {
+    // инициализация полей
+    this.name = name;
+    this.species = species;
+    this.gender = gender;
+    this.saying = saying;
+  }
+  // объявление метода toString()
+  // toString() возвращает строку, представляющую объект
+  // обращение к собственному атрибуту через this
+  toString() {
+    return `${this.name}; ${this.species}; ${this.gender}; ${this.saying}; `;
+  }
+}
+
+// создадим два класса наследующие от родительского часть свойств
+class Human extends WorldInhabitant {
+  constructor(name, species, gender, saying, legs, hands) {
```

Comment:
Why would `Human` get `species`, `legs`, and `hands` here? I believe all humans are humans and have a pretty default set of hands/legs.